Wrappers/Python/ccpi/viewer/viewerLinker.py

```diff
@@ -311,6 +313,19 @@ def __call__(self, interactor, event):
                 if (self.sourceVtkViewer.getSliceOrientation() != self.targetVtkViewer.getSliceOrientation()):
                     shouldPassEvent = False
 
+        # Slice from the slider
+        if event == "NoEvent" and self.linkSlice:
+            # print (f"Event: {event} SLIDER_EVENT: {SLIDER_EVENT}")
+            # guessing that it is a slider event
```

Comment:
```suggestion
            # Although I invoked a SLIDER_EVENT in widgets/slider.py, I noticed that a 
            # NoEvent is actually emitted/received. Therefore, we will link the slice in case of a 
            # NoEvent and self.linkSlice=True
```